lib/DDGC/Config.pm

```diff
@@ -5,7 +5,7 @@ use Moose;
 use File::Path qw( make_path );
 use File::Spec;
 use File::ShareDir::ProjectDistDir;
-use DDGC::Static;
+eval 'use DDGC::Static';
```

Comment:
I don't - don't care.

It's supposed to be the community-platform CSS bundle but it's either under-utilised or over-utilised - either way we don't care about the result of this.